src/ngraph_encapsulate_op.cc

```diff
@@ -515,9 +516,8 @@ class NGraphEncapsulateOp : public OpKernel {
           << "NGraphEncapsulateOp::Compute call starting for cluster "
           << m_ngraph_cluster;
       try {
-        op_backend->call(op_backend->compile(ng_function), ng_outputs,
-                         ng_inputs);
-
+        auto exec = op_backend->compile(ng_function);
```

Comment:
Find out if we can save the compiled function instead of compiling every time we call.